build-stack.sh

```diff
@@ -3,45 +3,40 @@
 # Script for building Kythe index of Haskell packages.
 # Usage: bash build-stack.sh /tmp/logs package...
 
-fail() {
-  (>&2 echo "$1")
+if (($# < 2)); then
+  echo "Usage: $0 /tmp/logs package..." >&2
   exit 1
-}
+fi
+
+if [[ -z "$STACK_YAML" ]]; then
```

Comment:
Please remove this check - I'm going to merge Travis support, and found it better to have a default stack.yaml anyway.